packages/e2e-tests/specs/experiments/multi-entity-editing.test.js

```diff
@@ -50,8 +51,8 @@ const createTemplatePart = async (
 	await page.keyboard.type( templatePartName );
 };
 
-const editTemplatePart = async ( textToAdd, isNested = false ) => {
-	await page.click(
+const editTemplatePart = async ( textToAdd, isNested = false, p = page ) => {
```

Comment:
this extra arg is useless right? I'm not saying where it's being passed.

Reply:
Yes, removed it.